.travis.yml

```diff
@@ -19,7 +19,7 @@ script:
   # check for changelog ReST compliance
   - rst2html.py --halt=warning changelog.rst >/dev/null
   # check for pep8 errors, only looking at branch vs master. If there are errors, show diff and return an error code.
-  - pep8radius master --docformatter --error-status || ( pep8radius master --docformatter --diff; false )
+  - pep8radius master --docformatter --diff --error-status
```

Comment:
Hi @bersace, this is not going to work. That `||` condition you removed was for the purpose of displaying what pep8radius is unhappy about, in case of it being unhappy.